lib/common/route_service_adapter.js

```diff
@@ -0,0 +1,16 @@
+import R from 'ramda';
+import Logger from './logger';
+
+export default function routeServiceAdapter(serviceName, functionName, ...argAdapters) {
+    return async function (ctx) {
+        const service = ctx.container[serviceName];
+
+        if (R.isNil(service)) {
+            Logger.error('Invalid service name', {serviceName, functionName});
+            ctx.status = 500;
+        } else {
```

Comment:
Else isn't necessary. Consider 
```suggestion
if (R.isNil(service)) {
            Logger.error('Invalid service name', {serviceName, functionName});
            ctx.status = 500;
            return;
        }
```
Thoughts?

Reply:
Oh yeah, you're totally right. tbh, this bit of code I copied over from SN. 😛

Reply:
@kjavaherpour addressed this in my latest commit